### PR TITLE
DS-765-RadioCardGroup jumping because of the border

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/RadioCard/index.tsx [hunk 1]
```diff
@@ -68,11 +68,7 @@ export const RadioCard = forwardRef<HTMLInputElement, RadioCardProps>(
           "flex min-w-fit cursor-pointer items-center rounded-radioCard-radiusBase border bg-radioCard-base p-radioCard-paddingLabel [&:has(:focus-visible)]:ring-4 [&:has(:focus-visible)]:ring-radioCard-base-focused",
           positionVariantStyles[radioPosition],
           alignVariantStyles[align],
-          {
-            "!border-2": disabled && checked,
-            "hover:shadow-radioCard": !disabled,
-          },
-
+          disabled && "hover:shadow-radioCard",
```
Comment (monikaracka) [inline packages/design-system/src/components/RadioCard/index.tsx:71]: What does this style do?  Hover on disabled?

**Review by monikaracka — COMMENTED** [review]
Screenshot 2023-08-25 at 08 34 06

And by the way it's something wrong with supporting text color when it's disabled.

File: packages/design-system/src/components/RadioCard/index.tsx [hunk 2]
```diff
@@ -68,11 +68,8 @@ export const RadioCard = forwardRef<HTMLInputElement, RadioCardProps>(
           "flex min-w-fit cursor-pointer items-center rounded-radioCard-radiusBase border bg-radioCard-base p-radioCard-paddingLabel [&:has(:focus-visible)]:ring-4 [&:has(:focus-visible)]:ring-radioCard-base-focused",
           positionVariantStyles[radioPosition],
           alignVariantStyles[align],
-          {
-            "!border-2": disabled && checked,
-            "hover:shadow-radioCard": !disabled,
-          },
-
+          disabled && checked && "border-2",
```
Comment (monikaracka) [inline packages/design-system/src/components/RadioCard/index.tsx:71]:
```suggestion
          checked && "border-2",
```

**Review by monikaracka — APPROVED** [review]

File: packages/design-system/src/components/RadioCard/index.tsx (hunk 2 above)
Comment (zuzannanap) [reply packages/design-system/src/components/RadioCard/index.tsx:71]: it's ok, i wanted it to be thick when disabled and checked at the same time

**Pull request merged**